test/CMakeLists.txt

```diff
@@ -236,13 +155,13 @@ test_headers(migraphx ${CMAKE_SOURCE_DIR}/src/include/migraphx/*.hpp)
 test_headers(migraphx/ref ${CMAKE_SOURCE_DIR}/src/targets/ref/include/migraphx/ref/*.hpp)
 
 if(MIGRAPHX_ENABLE_GPU)
-    test_headers(migraphx/gpu ${CMAKE_SOURCE_DIR}/src/targets/gpu/include/migraphx/gpu/*.hpp)
+    test_headers(migraphx/gpu HEADERS ${CMAKE_SOURCE_DIR}/src/targets/gpu/include/migraphx/gpu/*.hpp DEPENDS migraphx_gpu)
 endif()
 if(MIGRAPHX_ENABLE_CPU)
-    test_headers(migraphx/cpu ${CMAKE_SOURCE_DIR}/src/targets/cpu/include/migraphx/cpu/*.hpp)
```

Comment:
Right now, I dont use that, since it wont check for windows.h currently. I plan to add that to rocm-cmake later on and then switch this over to use `rocm_test_headers`.

Reply:
Seperate PR for this I'm assuming then @pfultz2  or were you going to do that change here too?

Reply:
Not here, in a separate PR.

Reply:
Got it